### ISSUE TITLE
Setup CI with Oracle DB

### ISSUE THREAD

**Issue by jgebal** [utPLSQL/utPLSQL-java-api#1]
We need to setup the project so that it can run on Travis CI.
The project should:
- [x] have java as main language
- [x] download oracle db (docker image?)
- [x] run oracle db
- [x] download latest (or specified) release of utPLSQL
- [x] install utplsql

**Comment by viniciusam** [comment]
The versioning schema it's kinda a problem. I had to declare two vars, one with the version and another with the build number. Take a look:
https://github.com/viniciusam/utPLSQL-java-api/blob/develop/.travis.yml
https://github.com/viniciusam/utPLSQL-java-api/blob/develop/.travis/install_utplsql.sh

I could clone the repository tag, but I think people would prefer to use the zip files install utPLSQL.

I installed the utPLSQL directly on the container, because I didn't want to add the complexity to download the sqlcl on the build right now, but that could change later.
